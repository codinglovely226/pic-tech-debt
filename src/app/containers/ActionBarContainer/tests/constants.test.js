import {
  TOGGLE_NEW_CALL_MODAL,
  UPDATE_NEW_CALL_MODAL_CALLER_LOCATION,
  UPDATE_NEW_CALL_MODAL_CALLER_ORGANISATION,
  UPDATE_NEW_CALL_MODAL_CENTRE_NAME,
  UPDATE_NEW_CALL_MODAL_FILTER,
  UPDATE_NEW_CALL_MODAL_ID,
  UPDATE_NEW_CALL_MODAL_NAME,
  UPDATE_NEW_CALL_MODAL_SELECTED,
  UPDATE_NEW_CALL_MODAL_SEX,
  UPDATE_NEW_CALL_MODAL_POSTCODE,
  UPDATE_NEW_CALL_MODAL_TIME,
} from '../constants';

describe('ActionBarContainer action constants', () => {
  it('TOGGLE_NEW_CALL_MODAL', () => {
    expect(TOGGLE_NEW_CALL_MODAL).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_CALLER_LOCATION', () => {
    expect(UPDATE_NEW_CALL_MODAL_CALLER_LOCATION).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_CALLER_ORGANISATION', () => {
    expect(UPDATE_NEW_CALL_MODAL_CALLER_ORGANISATION).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_CENTRE_NAME', () => {
    expect(UPDATE_NEW_CALL_MODAL_CENTRE_NAME).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_FILTER', () => {
    expect(UPDATE_NEW_CALL_MODAL_FILTER).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_ID', () => {
    expect(UPDATE_NEW_CALL_MODAL_ID).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_NAME', () => {
    expect(UPDATE_NEW_CALL_MODAL_NAME).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_SELECTED', () => {
    expect(UPDATE_NEW_CALL_MODAL_SELECTED).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_SEX', () => {
    expect(UPDATE_NEW_CALL_MODAL_SEX).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_POSTCODE', () => {
    expect(UPDATE_NEW_CALL_MODAL_POSTCODE).toMatchSnapshot();
  });
  it('UPDATE_NEW_CALL_MODAL_TIME', () => {
    expect(UPDATE_NEW_CALL_MODAL_TIME).toMatchSnapshot();
  });
});
