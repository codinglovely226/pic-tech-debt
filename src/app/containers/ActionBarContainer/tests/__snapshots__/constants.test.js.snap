exports[`ActionBarContainer action constants TOGGLE_NEW_CALL_MODAL 1`] = `"pic/ActionBarContainer/TOGGLE_NEW_CALL_MODAL"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_CALLER_LOCATION 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_CALLER_LOCATION"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_CALLER_ORGANISATION 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_CALLER_ORGANISATION"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_CENTRE_NAME 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_CENTRE_NAME"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_FILTER 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_FILTER"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_ID 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_ID"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_NAME 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_NAME"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_POSTCODE 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_POSTCODE"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_SELECTED 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_SELECTED"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_SEX 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_SEX"`;

exports[`ActionBarContainer action constants UPDATE_NEW_CALL_MODAL_TIME 1`] = `"pic/ActionBarContainer/UPDATE_NEW_CALL_MODAL_TIME"`;
